### DISCUSSION TITLE
Will WiFi models be supported (or are they already)?

### DISCUSSION THREAD

**Discussion by AleXSR700** [pvvx/THB2#26] [Q&A]
Hi everyone,

with the new Xiaomi Mijia firmware not being flashable anymore and the risk increasing that Xiaomi devices won't be flashable in the future, I am looking into Tuya devices.

However, Tuya is not known for being open to the community either.

So while I am looking at the different Tuya devices I saw that the WiFi versions of the different Temp & Humidity sensors are just as cheap and essentially identical externally.

Does anybody know if they are using the same basic chips and if they can also be flashed with an alternative firmware that will allow data access purely locally (i.e. without Tuya cloud etc.)?

Here is an example on Ali
https://a.aliexpress.com/_EJ6P9Mh

Thank you for your help.

Alex

**Comment by pvvx** [comment]
Tuya Wi-Fi devices consume too much battery power. This is not an option to use.


Example: 
Door Window Sensor 
Battery: 2xAAA 

Standby current consumption:
![image](https://github.com/pvvx/THB2/assets/12629515/2d07e66c-32de-4735-9611-1c0c90d1150b)

When triggered:
![image](https://github.com/pvvx/THB2/assets/12629515/d2b4402a-f769-45f8-a3c1-2d8d21194d07)

It won't start with salt batteries. Only with lithium, since peak currents are up to 500 mA.

PS: Tuya battery-powered WiFi sensors should be thrown in the trash immediately.

**Comment by AleXSR700** [comment]
Oh, okay, I understand. But are equivalent Tuya bluetooth devices better (the ones using AAA batteries) or is it a Tuya chip issue?
